### DISCUSSION TITLE
[Suggestion] Racial-based Collections

### DISCUSSION THREAD

**Discussion by lhsaffar** [xivdev/Penumbra#201] [Ideas]
With the more stable metadata in the current Alpha of Penumbra, I'd like to suggest the possibility for collections to have "racial" criteria.

As it stands right now, it would take a lot of work to have non-chara collection ppl not share body types between different "shared" races ie Midlander vs Miqo'te vs Elezen.

With the new (alpha) Inheritance system, a character could still have their own collection independantly from any racial ones, or they could directly inherit their race's collection first before any chara-specific changes are applied.

Noting that the implementation is apparently possible , seeing that for an actor, "the race is known by the draw object itself".

**Comment by polaerise** [comment]
Those would be so amazing!
